### ISSUE TITLE
[2023] Loss Landscapes are All You Need: Neural Network Generalization Can Be Explained Without the Implicit Bias of Gradient Descent

### ISSUE THREAD

**Issue by gatheluck** [gatheluck/PaperReading#535]
## 論文リンク
- [OpenReview](https://openreview.net/forum?id=QC10RmRbZy9)

## 公開日（yyyy/mm/dd）
2023/02/02
ICLR 2023

### Research Question

研究で明らかにしたい問を端的に表したもの.

### Elevator Pitch

[潜在的なニーズを満たしたり、潜在的な課題を解決したり] したい
[対象ユーザー] 向けの, 
[提案手法] という手法は, 
[提案手法のカテゴリー] です.
これは [提案手法の出来ること] ができ, 
[代替手段のSoTA] とは違って, 
[差別化の決定的な特徴]
が備わっている.

## TeX

```
% yyyy/mm/dd
@inproceedings{
    chiang2023loss,
    title={Loss Landscapes are All You Need: Neural Network Generalization Can Be Explained Without the Implicit Bias of Gradient Descent},
    author={Ping-yeh Chiang and Renkun Ni and David Yu Miller and Arpit Bansal and Jonas Geiping and Micah Goldblum and Tom Goldstein},
    booktitle=ICLR,
    year={2023}
}
```
